.eslintrc

```diff
@@ -0,0 +1,31 @@
+{
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es6": true,
+        "node": true
+    },
+    "extends": "eslint:recommended",
```

Comment:
That's okay. Atleast now we are positive that the config we are currently using is the best suited for out needs. Merge then whenever you are ready. Let's start working on Travis regarding this and we will soon add testing to Travis as well.